live-site/components/welcome-section/WelcomeSection.styles.ts

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import { max } from '../../../shared-ui/lib/responsive';
+import { colors } from '../../../shared-ui/style/colors';
+import { P } from '../../../shared-ui/style/typography';
+
+const StyledWelcomeSectionContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-conetnt: center;
+  @media ${max.tablet} {
+    flex-direction: column;
+    align-items: center;
+  }
+`;
+
+const StyledRaceContainer = styled.div`
+  background-color: ${colors.TEXT_BOX};
+  width: 40em;
+  height: 28.5em;
+  border-radius: 1.5em;
+  margin: 1.3em;
```

Comment:
hm
<img width="1233" alt="Screenshot 2023-01-05 at 5 25 35 PM" src="https://user-images.githubusercontent.com/59738880/210891896-0b62089c-a721-4f3d-a8dd-19fbbaec562b.png">
mm ok so right now it looks like this at certain screen sizes 

To fix this, can we change this to be the following so that on larger screensizes there's no right margin (since we'll apply that on the larger container to make both sides even). And on smaller screensizes we don't need margin on any of the other sides besides the top to add some spacing there

`  margin: 1.3em 0 1.3em 1.3em;
  @media ${max.tablet} {
    margin: 1.3em 0 0 0;
    width: 90%;
  }`